charts/images.yaml

```diff
@@ -191,9 +191,9 @@ images:
   tag: "v1.2.0"
   targetVersion: ">= 1.22"
 - name: ingress-default-backend
-  sourceRepository: github.com/gardener/ingress-default-backend
-  repository: eu.gcr.io/gardener-project/gardener/ingress-default-backend
-  tag: "0.10.0"
+  sourceRepository: https://github.com/kubernetes/ingress-gce
+  repository: k8s.gcr.io/defaultbackend-amd64
```

Comment:
Isn't there 1.5 tag for a multi-arch image https://console.cloud.google.com/gcr/images/google-containers/global/defaultbackend? Is the multiarch image deprecated?

Reply:
I don't see 1.5 tag for multi-arch, so it seems to be deprecated or no longer maintained. This is not the only image of this kind (cc @acumino @shafeeqes), so we will need to have some logic in our image vector for this anyways when we get to supporting non-`amd64` archs.

Reply:
dear collegues. for gardener-cicd we allow a "relaxed semver" flavour, with the following deviations from "strict" semver as defined on semver.org:

- allow optional `v`-prefix
- allow omitting patch-component

To still be able to perform semver-arithmetics, we normalise to strict semver:

- strip `v` prefix, if present
- append `.0` as patchlevel if absent (e.g. `1.5` would be interpreted as `1.5.0` in your example)